frontend/src/components/SessionBoard.js

```diff
@@ -0,0 +1,28 @@
+import styled from 'styled-components/macro';
+import SessionTile from './SessionTile';
+
+export default function SessionBoard({ sessions }) {
+    return (
+        <>
+            <Wrapper>
+                {sessions.map((session) => (
+                    <li key={session.id}>
+                        <SessionTile session={session} />
+                    </li>
+                ))}
+            </Wrapper>
+        </>
+    );
+}
+
+const Wrapper = styled.ul`
+    display: grid;
+    justify-content: center;
+    height: 100%;
+    background: none;
+    overflow-y: scroll;
```

Comment:
Ah, refering to my comment in `PageLayout`, making the main scroll would be more flexible